src/Components/PairingForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { findGame } from '../redux/actions/boardGame';
+import { setLiquorType, setRandomDrink } from '../redux/actions/liquorSearch'
+
+const PairingForm = () => {
+
+    const dispatch = useDispatch()
+
+    const games = useSelector(state => state.boardGame.allGamesData.games)
+    const type = useSelector(state => state.liquorSearch.liquorSearchWord)
+
+    const gameNames = games.map((game) => {
+        return (
+            <option key={game.id} value={game.name} />
+        )
+    })
+
+    const findRandomDrink = (event) => {
+        event.preventDefault()
+        fetch(`https://www.thecocktaildb.com/api/json/v1/1/filter.php?i=${type}`)
+            .then(res => res.json())
+                .then(data => {
+                    console.log('data', data)
```

Comment:
I encourage you to start making sure your PRs are production ready. Out in the wild, many reviewers will reject PRs that have console logs or comments. This is because there's rarely any step after the PR step where these items can be removed. It'll look really professional if you submit super clean PRs to your employers, so it's good to get into the practice now.

Reply:
Thanks for that feedback!!